charts/opentelemetry-operator/values.yaml

```diff
@@ -42,7 +42,7 @@ manager:
     tag: ""
   collectorImage:
     repository: ""
-    tag: 0.111.0
+    tag: 0.112.0
```

Comment:
FYI there is a bug with collector 0.112.0 and the hostmetricsreceiver on k8s: https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/35990

Reply:
argh, i assume there won't be a patch for that right? Should i call this out in the release notes or something?

Reply:
the fix will be in v0.113.0

Reply:
kk we may as well skip then too...

Reply:
Is there a downside to updating this PR to use v0.113.0 of the collector when it releases this week?

Reply:
nope, we can do that.